models/apple_search_ads.yml

```diff
@@ -2,182 +2,256 @@ version: 2
+        description: The sum of new_downloads and redownloads.
       - name: impressions
-        description: The number of impressions of keyword in a given day
+        description: The number of impressions of keyword in a given day.
       - name: spend
-        description: The spend on keyword in the given day
+        description: The spend on keyword in the given day.
 
   - name: apple_search_ads__organization_report
-    description: Each record represents the daily ad performance of each organization
+    description: Each record represents the daily ad performance of each organization.
     tests: 
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - organization_id
             - date_day
     columns:
       - name: organization_id
-        description: Organization ID associated with this record
+        description: Organization ID associated with this record.
+        tests:
+          - not_null
       - name: organization_name
-        description: Organization name associated with this record
+        description: Organization name associated with this record.
       - name: date_day
-        description: Date of the report
+        description: Date of the report.
+        tests:
+          - not_null
       - name: currency
-        description: This currency value should match the respective organization's currency value
+        description: This currency value should match the respective organization's currency value.
       - name: taps
-        description: Number of taps on organization on given day
+        description: Number of taps on organization on given day.
       - name: new_downloads
-        description: App downloads from new users who have never before installed app of organization in a given day
+        description: App downloads from new users who have never before installed app of organization in a given day.
       - name: redownloads
-        description: Number of user downloads where user deletes app and downloads the same app again following a tap on an ad on the App Store, or downloads the same app to an additional device of organization in a given day 
+        description: Number of user downloads where user deletes app and downloads the same app again following a tap on an ad on the App Store, or downloads the same app to an additional device of organization in a given day.
       - name: total_downloads
-        description: The sum of new_downloads and redownloads
+        description: The sum of new_downloads and redownloads.
       - name: impressions
-        description: The number of impressions of organization in a given day
+        description: The number of impressions of organization in a given day.
       - name: spend
-        description: The spend on organization in the given day  
+        description: The spend on organization in the given day.
   
   - name: apple_search_ads__search_term_report
```

Comment:
It looks like the `match_type` field documentation was missed for this model.
![image](https://user-images.githubusercontent.com/74217849/180824801-66116b09-2818-4833-a412-02a778a2fb25.png)